src/program/operations/save/index.js

```diff
@@ -37,20 +50,50 @@ const shouldUpdateScript = scriptName => {
   });
 };
 
+/**
+ * Save script to config
+ *
+ * @param {string} scriptName - name of script to save to config
+ */
+const saveScript = scriptName => {
+  GlobalConfig.addScriptName(scriptName);
+  GlobalConfig.save();
+  printMessage(formatMessage(messages.savedScript, { scriptName }));
+};
+
+/**
+ * Delete script file
+ *
+ * @param {string} scriptName - name of script to delete
+ */
+const deleteScript = scriptName => {
+  deleteYamlFile(`${SCRIPTS_DIRECTORY_PATH}/${scriptName}.simple.yml`);
+  deleteYamlFile(`${SCRIPTS_DIRECTORY_PATH}/${scriptName}.yml`);
+  deleteJsonFile(`${SCRIPTS_DIRECTORY_PATH}/${scriptName}.simple.json`);
+  deleteJsonFile(`${SCRIPTS_DIRECTORY_PATH}/${scriptName}.json`);
+};
+
+/**
+ * Run the save operation
+ *
+ * @param {string[]} args - arguments passed to the save operation
+ */
 const run = args => {
-  const yamlFileName = args[0];
+  const fileName = args[0];
 
-  if (isValidYamlFileName(yamlFileName)) {
-    const yamlFile = loadYamlFile(yamlFileName);
+  if (isValidYamlFileName(fileName)) {
+    const yamlFile = loadYamlFile(fileName);
     const scriptName = getFirstKey(yamlFile);
 
     GlobalConfig.load();
     if (GlobalConfig.hasScript(scriptName)) {
       // Check if user wants to update the script
       shouldUpdateScript(scriptName).then(answer => {
         if (answer) {
+          // Delete previous script
```

Comment:
nit:
- [ ] `Delete previous script` => `Delete any previously saved scripts with the same name`